pkg/controller/plan/kubevirt.go

```diff
@@ -704,6 +712,27 @@ func (r *KubeVirt) virtualMachine(vm *plan.VMStatus) (object *cnv.VirtualMachine
 		return
 	}
 
+	//If the VM name is not valid according to DNS1123 labeling
+	//convention it will be automatically changed.
+	var originalName string
+
+	if errs := k8svalidation.IsDNS1123Label(vm.Name); len(errs) > 0 {
+		originalName = vm.Name
+
+		generatedName := changeVmName(vm.Name, vm.ID)
+		nameExist, errName := r.checkIfVmNameExist(generatedName)
+		if errName != nil {
+			err = liberr.Wrap(errName)
+			return
+		}
+		if nameExist {
+			generatedName = generatedName + "-" + vm.ID[:4]
+		}
+		vm.Name = generatedName
+		r.Log.Info("VM name was not compatible with DNS1123 RFC, changing to ", "vm",
```

Comment:
```suggestion
		r.Log.Info("VM name was incompatible with DNS1123 RFC, changing to ", "vm",
```
and what's the reason for `"vm"` here? did you mean to specify both the original name and the generated name?

Reply:
Yes, but I saw that the original name parameter is missing and probably was accidentally deleted in one of the recent changes, adding it back